pipeline/Makefile

```diff
@@ -296,6 +299,14 @@ data/extracted/cta_train_ridership.csv:
 		--json_url="$(PORTAL_CTA_TRAIN)" \
 		--soql_file="extract/cta_train_ridership.sql" \
 		--output_file="data/extracted/cta_train_ridership.csv"
+		
+
+
+data/extracted/daily_rideship.csv:
```

Comment:
let's distinguish the ridership data sets!  so either do daily_bus_ridership or do something like cta_bus_ridership